packages/stockflux-components/src/components/glyphs/arrows/priceArrowDown.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="9px" height="7px" viewBox="0 0 9 7" version="1.1" 
+    xmlns="http://www.w3.org/2000/svg" 
+    xmlns:xlink="http://www.w3.org/1999/xlink">        
+    <g id="Assets" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <path fill="currentColor" d="M4.5,0 L9,7 L0,7 L4.5,0 Z" transform="translate(4.500000, 3.500000) scale(1, -1) translate(-4.500000, -3.500000) "></path>
```

Comment:
Nitpick; does there need to be a space at the end of the `transform` variable?

Reply:
removed and still works correctly.